### PR TITLE
commit for UXPLATFROM-8603

### PR DESCRIPTION
**Pull request by JRPOOJA**
### Summary

Closes #

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by CristinaERoman — APPROVED** [review]
The changes look good from my side.

**Review by PBKarthik — APPROVED** [review]

**Review by razvanluca — APPROVED** [review]
looks good for me

**Review by CristinaERoman — APPROVED** [review]

File: packages/terra-application-utility/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 2.51.0 - (April 4, 2023)
```
Comment (sdadn) [inline packages/terra-application-utility/CHANGELOG.md:5]: These headings are automatically added when we do a release; for now we can keep them under the `unreleased` section.

```suggestion
```

**Comment by sdadn** [comment]
Can the title be update to be more descriptive of the code change?

**Review by supreethmr — APPROVED** [review]

**Pull request merged**